src/db2tclsh.c

```diff
@@ -20,6 +20,10 @@
 #include "tcl.h"
 #include "db2tcl.h"
 
+#if TCL_MAJOR_VERSION >= 9
+int Tcl_AppInit(Tcl_Interp *);
```

Comment:
According to the docs, this prototype should be provide by `<tcl.h>`.  Redundant?

Reply:
Using same solution as described here https://github.com/python/cpython/issues/104363 we don't actually use db2tclsh in HammerDB but it is a built in shell that we leave as part of the build in case it is useful.